### DISCUSSION TITLE
为什么download zip code文件到本地时候，window防火墙会报main.zip文件中有木马文件？

### DISCUSSION THREAD

**Discussion by RyanBin0126** [GaiZhenbiao/ChuanhuChatGPT#465] [General]
![image](https://user-images.githubusercontent.com/58510134/228734101-e1876c86-df95-4e0d-8891-d2b1ece9caa8.png)

**Comment by GaiZhenbiao** [comment]
这，我也不知道哇。**请确保zip文件是从GitHub上的这个repo下载**。如果就是这样，忽略这个警告吧。

**Comment by Keldos-Li** [comment]
为啥是main.zip，不应该是ChuanhuChatGPT-main.zip类似的文件名吗……

开源项目不放心可以自行审查源码。

**Comment by Keldos-Li** [comment]
参考这个：https://github.com/gradio-app/gradio/issues/3230

**Comment by RyanBin0126** [comment]
好的，谢谢@GaiZhenbiao @Keldos-Li
